azure/blobs_logs_monitoring/index.js

```diff
@@ -2,191 +2,613 @@
 // under the Apache License Version 2.0.
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2021 Datadog, Inc.
+var https = require('https');
 
-var tls = require('tls');
-
-const VERSION = '0.2.0';
+const VERSION = '1.0.0';
```

Comment:
Is the http version requirement coming from Azure? Any reason why we do not use http 2?